### ISSUE TITLE
Unfollowing works but follower isn't removed from followers collection

### ISSUE THREAD

**Issue by fauno** [hyphacoop/social.distributed.press#27]
Running a local fediverse I can follow sites on the Social Inbox but unfollowing doesn't remove the follower:

```ruby
base_endpoint = "/v1/@skel@skel.sutty.local"
client.get(endpoint: "#{base_endpoint}/followers")
=> {"@context"=>"https://www.w3.org/ns/activitystreams",
 "type"=>"OrderedCollection",
 "id"=>"https://dpress.sutty.local/v1/@skel@skel.sutty.local/followers",
 "items"=>["https://ktistec.sutty.local/actors/suttier", "https://mastodon.sutty.local/users/fauno"],
 "totalItems"=>2}
```

The `items` array should be empty because I unfollowed from both instances.

**Comment by fauno** [comment]
Related to #47

**Comment by sutty-coop** [comment]
ping for @RangerMauve or @akhileshthite if you find the time to check this out

**Comment by akhileshthite** [comment]
> ping for @RangerMauve or @akhileshthite if you find the time to check this out

Hey @sutty-coop, I think this was addressed in #47 so we can close this now?
